.github/workflows/catalog_consistency.yml

```diff
@@ -25,9 +25,12 @@ jobs:
      - uses: actions/setup-python@v5
        with:
          python-version: '3.9'
-         
-     - run: curl -LsSf https://astral.sh/uv/install.sh | sh
-     - run: uv pip install --system -e ".[tests]"
+         cache: 'pip' # caching pip dependencies
```

Comment:
This should be returned, as it was a change done by Elron to us uv instead of pip. Otherwise, we should consult @elronbandel .

Reply:
this is just a merge thing. It should be as in the master